proselint/command_line.py

```diff
@@ -95,15 +97,25 @@ def print_errors(filename, errors, output_json=False, compact=False):
 @click.option('--time', '-t', is_flag=True, help="Time on a corpus.")
 @click.option('--demo', is_flag=True, help="Run over demo file.")
 @click.option('--compact', is_flag=True, help="Shorten output.")
+@click.option('--dump-config', is_flag=True, help="Prints current config.")
+@click.option('--dump-default-config', is_flag=True,
+              help="Prints default config.")
 @click.argument('paths', nargs=-1, type=click.Path())
 @close_cache_shelves_after
-def proselint(paths=None, config=None, version=None, clean=None, debug=None,
-              output_json=None, time=None, demo=None, compact=None):
+def proselint(paths=None, config=None, version=None, clean=None,
+              debug=None, output_json=None, time=None, demo=None, compact=None,
+              dump_config=None, dump_default_config=None):
     """Create the CLI for proselint, a linter for prose."""
+    if dump_default_config:
+        return print(json.dumps(default, sort_keys=True, indent=4))
+
+    config = load_options(config, default)
+    if dump_config:
+        return print(json.dumps(config, sort_keys=True, indent=4))
```

Comment:
There's something mildly grating about returning the `None` output of `print`. Not a big deal, and I can't really explain the feeling, but I prefer the original.

Reply:
I get you, honestly. It's just a shorter more concise way to return - but, you're right, perhaps we shouldn't return from a function call that evaluates to a void expression and instead just use `return` for clarity.